hack/docs/internal/cronjob-tutorial/controller-implementation.go

```diff
@@ -0,0 +1,528 @@
+package cronjob
+
+
+const ControllerIntro = `
+// +kubebuilder:docs-gen:collapse=Apache License
+
```

Comment:
Nit, could we have an standard for the files name?
Can we use _ instead of - in all of them?
Also, could we not have the sufix test since it is a reserved in golang for the tests ?

Reply:
Thx for pointing out 👍